soh/soh/Enhancements/mods.cpp

```diff
@@ -1018,6 +1018,39 @@ void RegisterRandomizerSheikSpawn() {
     });
 }
 
+void RegisterRandomizedEnemySizes() {
+    GameInteractor::Instance->RegisterGameHook<GameInteractor::OnActorInit>([](void* refActor) {
+        // Randomized Enemy Sizes
+        Player* player = GET_PLAYER(gPlayState);
+        Actor* actor = static_cast<Actor*>(refActor);
+
+        // Only apply to enemies and bosses. Exclude the wobbly platforms in Jabu because they need to act like platforms.
+        if (!CVarGetInteger("gRandomizedEnemySizes", 0) || (actor->category != ACTORCAT_ENEMY && actor->category != ACTORCAT_BOSS) || actor->id == ACTOR_EN_BROB) {
+            return;
+        }
+
+        float randomNumber;
+        float randomScale;
+
+        uint8_t bigActor = rand() % 2;
```

Comment:
I've seen pushback on PRs calling `rand()` a few times but for something like this I think `rand()` works perfectly. There are other "on the fly" random things that use `rand()` throughout the codebase - random fuse timer for bombs for example

https://github.com/HarbourMasters/Shipwright/blob/02f61bc3faa280581fc17fc9f76984fa9c52a1ee/soh/src/overlays/actors/ovl_En_Bom/z_en_bom.c#L105-L107

Reply:
My next project is to get rid of ALL rand() calls, so those existing ones are living on borrowed time :)  What I'd prefer you to do here is use the Random function from 3drando if for no other reason than this is a C++ file, so we should be using C++ functionality, although no matter what you do here, it will be changed by me soon anyway.  The 3drando header is already included in this file, so it should be a trivial substitution.